.pre-commit-config.yaml

```diff
@@ -22,20 +22,28 @@ repos:
         additional_dependencies: [sphinx]
         args: ["--config", "docs/rstcheck.cfg"]
 
-  - repo: https://github.com/pycqa/pydocstyle
-    rev: 6.1.1
-    hooks:
-      - id: pydocstyle
-        additional_dependencies: [toml]
-
   - repo: local
     hooks:
-      - id: pylint
-        name: pylint
-        entry: pylint
+      - id: pylama
+        name: pylama
+        entry: pylama
         language: system
         types: [python]
-        args: [
-            "-rn", # Only display messages
-            "-sn", # Don't display the score
-          ]
+
+  - repo: https://github.com/PyCQA/bandit
+    rev: "1.7.4"
+    hooks:
+      - id: bandit
+        files: "req2flatpak.py"
+        args:
+          - "--aggregate"
+          - "vuln"
+          - "-ll"
+          - "-q"
+          - "-f"
+          - "custom"
+          - "--msg-template"
```

Comment:
out of interest... why a custom message template? (my gut feeling is that it may be better to rely on default message templates to avoid having to maintain and debug the template in the future)

Reply:
The default output includes run metrics and long, elaborate issue descriptions. I think that this is impractical for use as a pre-commit hook. The default template for the `custom` format option is too hard to read.

Reply:
ok alright, thanks for explaining!